Readme.md

```diff
@@ -0,0 +1,48 @@
+2. Next, [create an OpenID Connect Native app](https://help.okta.com/en/prev/Content/Topics/Apps/Apps_App_Integration_Wizard.htm) with a Redirect URI value of ``http://127.0.0.1:[available_port]``, with the ``available_port`` value being a port available on your machine (so that your console app can listen for the browser response on that port. __Important note__: don't forget to assign at least one user to your new OpenID Connect app!
+3. Open the ``Okta OpenID Connect Windows Native Examples`` solution in Visual Studio 2015 and in the ``Okta OpenID Connect Console (Code Auth Flow - Native Browser)`` project, edit the __App.config__ file to set the following values:    
+     a. __``okta:OrganizationUrl``__: the full url of your Okta org (e.g. ``https://company.okta.com``)     
+     a. __``okta:AuthorizationServerUrl``__: the full url of your Okta org (e.g. ``https://company.okta.com``)   
+     b. __``okta:ClientId``__: the ``Client ID`` value of your Okta OIDC Native app.      
+     c. __``okta:RedirectUri``__: a valid redirect uri as set up in your Okta OIDC Native app. This value should be of the form ``http://127.0.0.1:{any_port}/`` and configured as a redirect uri in your Okta OIDC app. __Important note: Make sure to include the trailing slash!__
+     d. __``okta:Scopes``__: the OpendID Connect scopes your application will request from Okta - you can use the default scopes as already configured.    
+     e. __``okta:ResponseType``__: the OpenID Connect response type (can currently be ``cpde`` or ``code id_token``)
+4. You can test the application with the parameters above and verify that you can sign in with Okta in your browser (or leverage an existing Okta session). You should be able to verify that your console application is able to authenticate you with the same credentials you used in the browser.
+5. If you want to test the ability to call an external API (for instance, our [ASP.NET Core Web API sample](https://github.com/raphaellondner-okta/okta-oauth-dotnetcore-rs-simple)), you must have access to Okta's API Access Management product, which is currently in beta version. If you want access to this product, please submit a request on the [Okta beta site](https://oktabeta.zendesk.com) and select ``API Access Management`` in the _Beta Name_ dropdown list.
+6. Once you've been granted access to Okta's API Access Management product, navigate to __Security-->API__ in the Admin dashboard of your Okta organization. You should see a page similar to the screenshot below:
+     ![Authorization Servers Home Page](https://github.com/raphaellondner-okta/okta-oauth-nativewindows/blob/google-console/assets/Okta_AuthorizationServersHomePage.PNG)
+7. Press the ``Add Authorization Server`` button and  a descriptive name (such as _ToDo List API_), a resource Uri (such as _http://todolist.example.com_), as well as an optional description. You should see aenter page similar to the screenshot below:
```

Comment:
aenter -> not sure what this is supposed to be

As a general side note about submitting PR's - you should read through the code/documentation at least once before requesting a review. Generally shouldn't be catching spelling mistakes and bad formatting at review time.